##### Resonancia catolica

La résonance universelle est le rapport du pouvoir rotatoire (spin en anglais) = 0, +180° et moins 180°, que je simplie en -1 [0] 1.

L'espace est analogue à une gigantesque capacitance en résonance avec une gigantesque inductance de fréquence angulaire $\omega=\pi 2f$, analogue à $\pi 2 r$, en fait les modernes réinventent la roue en changeant la terminalogie.

In [1]:
L=1/n(pi)*360E-8;C=1/(n(pi)*360E+8)
w0=1/sqrt(L*C);w1=2*n(pi)*0.5

/home/pi/.local/lib/python3.8/site-packages/sympy/core/function.py:107: DeprecationWarning: inspect.getargspec() is deprecated since Python 3.0, use inspect.signature() or inspect.getfullargspec()
  evalargspec = inspect.getargspec(cls.eval)


In [2]:
w0,w1

(3.14159265358979e8, 3.14159265358979)

In [3]:
u=n(pi)*4E-7;c=299792458;s=1/(u*c*c)

In [4]:
wc=1/(sqrt(u*s));int(wc)

299792458

In [5]:
vc=wc/(2*n(pi));vc

4.77134515923694e7

In [6]:
wf=w0/(2*n(pi))

In [7]:
pr=wf/vc;pr

1.04792251097584

Le rapport fréquence angulaire de $\pi$ lumière sur fréquence angulaire $c$ donne une constante universelle dao = 道 = 1.04792251097584

In [9]:
deg(x)=x*360/(2*n(pi));deg(1)

57.2957795130823

In [11]:
deg(x)=x*180/(2*n(pi));deg(pr)

30.0207685678337

In [12]:
deg(x)=x*360/(2*n(pi));deg(pr)

60.0415371356674